Specs/Widgets/Viewer/ViewerSpec.js

```diff
@@ -1021,4 +1031,405 @@ defineSuite([
+        viewer = createViewer(container);
+
+        expect(function() {
+            viewer.flyTo();
+        }).toThrowDeveloperError();
+    });
+
+    it('flyTo flys to Cesium3DTileset with default offset when options not defined', function() {
+        viewer = createViewer(container);
+
+        var path = './Data/Cesium3DTiles/Tilesets/TilesetOfTilesets';
+        var tileset = new Cesium3DTileset({
+            url : path
+        });
+
+        // load tileset to test
+        return tileset.readyPromise.then(function() {
+            var promise = viewer.flyTo(tileset);
+            var wasCompleted = false;
+
+            spyOn(viewer.camera, 'flyToBoundingSphere').and.callFake(function(target, options) {
+                wasCompleted = true;
+                options.complete();
+            });
+
+            viewer.render();
+
+            return promise.then(function() {
+                expect(wasCompleted).toEqual(true);
+            });
+        });
+
+    });
+
+    it('flyTo flys to Cesium3DTileset with default offset when offset not defined', function() {
+        viewer = createViewer(container);
+
+        var path = './Data/Cesium3DTiles/Tilesets/TilesetOfTilesets';
+        var tileset = new Cesium3DTileset({
+            url : path
+        });
+
+        var options = {};
+
+        // load tileset to test
+        return tileset.readyPromise.then(function() {
+            var promise = viewer.flyTo(tileset, options);
+            var wasCompleted = false;
+
+            spyOn(viewer.camera, 'flyToBoundingSphere').and.callFake(function(target, options) {
+                wasCompleted = true;
+                options.complete();
+            });
+
+            viewer.render();
+
+            return promise.then(function() {
+                expect(wasCompleted).toEqual(true);
+            });
+
+        });
+    });
+
+    it('flyTo flys to target when target is Cesium3DTileset and options are defined', function() {
+        viewer = createViewer(container);
+
+        var path = './Data/Cesium3DTiles/Tilesets/TilesetOfTilesets';
+        var tileset = new Cesium3DTileset({
+            url : path
+        });
+
+        // load tileset to test
+        return tileset.readyPromise.then(function() {
+            var boundingSphere = tileset.boundingSphere;
+            var offsetVal = new HeadingPitchRange(3, 0.2, 2.3 * boundingSphere.radius);
+            var options = {
+                offset : offsetVal,
+                duration : 3.0,
+                maximumHeight : 5.0
+            };
+
+            var promise = viewer.flyTo(tileset, options);
+            var wasCompleted = false;
+
+            spyOn(viewer.camera, 'flyToBoundingSphere').and.callFake(function(target, options) {
+                wasCompleted = true;
+                options.complete();
+            });
+
+            viewer.render();
+
+            return promise.then(function() {
+                expect(wasCompleted).toEqual(true);
```

Comment:
Also expect `flyToBoundingSphere` [`toHaveBeenCalledWith`](https://jasmine.github.io/2.0/introduction.html#section-Spies) the proper options.